File: 김다은/0_Array&List-TimeComplexity/2562.cpp
#include<iostream>

using namespace std;

int main() {

	int tmpNum;
	int max;
	int index;

	for (int i = 0; i < 9; i++) {
		cin >> tmpNum;
		if (i == 0) { max = tmpNum; index = i + 1; }

		if (tmpNum > max) { max = tmpNum; index = i + 1; }
	}

	cout << max << endl;
	cout << index;
}

File: 박혜린/1주차/2562.py
i = 9
li = []
max_n = 0

while i > 0:
  n = int(input())
  if 0 < n < 100:
    li.append(n)
    i -=1
  else:
    print("Input a number between 0 and 100")

for i in li:
  if i > max_n:
    max_n = i

# print(max(li))
# print(li.index(max(li))+1)

print(max_n)
print(li.index(max_n)+1)
